imagedephi/utils/network.py

```diff
@@ -12,3 +13,14 @@ async def wait_for_port(port: int, host: str = "127.0.0.1") -> None:
             writer.close()
             await writer.wait_closed()
             return
+
+
+def unused_tcp_port() -> int:
+    with socket.socket() as sock:
+        sock = socket.socket()
```

Comment:
Just for my own understanding, wouldn't sock already be `socket.socket()` from line 19? I am unsure why line 20 is necessary.

Reply:
You're right. I think I was testing something and accidently committed this line. I'll fix it.